src/plugins/visualizations/public/types.ts

```diff
@@ -30,13 +27,6 @@ export type VisualizationControllerConstructor = new (
   vis: ExprVis
 ) => VisualizationController;
 
-export interface SavedVisState {
```

Comment:
All changes which you did in visualization are not related to data-table telemently. At least please add a couple of notes in description, or do it in a separate PR. Or probably you can add these changes to your #88317 where you already moved some types. 

I'm ok to see some lines in PR description

Reply:
PR description updated